docs/en/connector-v2/sink/Enterprise-WeChat.md

```diff
@@ -0,0 +1,53 @@
+# Enterprise WeChat
+
+> Enterprise WeChat sink connector
+
+## Description
+
+A sink plugin which use Enterprise WeChat robot send message
+> For example, if the data from upstream is [`"告警状态": "firing", "告警时间": "2022-08-03 01:38:49"，"告警内容": "磁盘使用超出阈值"`], the output content to WeChat Robot is the following:
```

Comment:
I suggest you change the chinese to english.

Reply:
> I suggest you change the chinese to english.

Thx,i have changed in new committ